history/cultures/gnoll_group.txt

```diff
@@ -0,0 +1,45 @@
+# Gnoll
```

Comment:
I suggested a different number of techs for the gnolls (and the 1 and 2 tech levels) They can't leave the tribal era, it's impossible due to the government.

Reply:
I dont see any suggestions on old sheet or new one, here is the link to what I am using right now:

https://docs.google.com/spreadsheets/d/18zS-FAte6nsejx7rpzSmVuioRXps3iWSvSkx_xRJSWI/edit#gid=165545728

I would suggest a culture tech for all level 1's and 2's that give them a base tech boost because they are so far behind everyone.

Reply:
> I dont see any suggestions on old sheet or new one, here is the link to what I am using right now:

I said what you missed:

> I suggested a different number of techs for the gnolls (and the 1 and 2 tech levels). They can't leave the tribal era, it's impossible due to the government. Tribes can't leave the tribal era.

> I would suggest a culture tech for all level 1's and 2's that give them a base tech boost because they are so far behind everyone.

I wouldn't. There's Exposure anyway.

Reply:
OK, so from T2 ---> T3 there will be a 15 tech gap with what I have to work with, I can make T1 have 11, and T2 will have 13.  Is that okay?

Reply:
I don't think exposure will make up for a 15 tech gap especially since each tech is like 20-30 years of development

Reply:
> OK, so from T2 ---> T3 there will be a 15 tech gap with what I have to work with, I can make T1 have 11, and T2 will have 13. Is that okay?

Yes.

Reply:
> I don't think exposure will make up for a 15 tech gap especially since each tech is like 20-30 years of development

It def will take a while, but there's a giant tech gap.